projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -160,6 +160,11 @@ public static function load_wpcom_user_features() {
 		require_once __DIR__ . '/features/wpcom-sidebar-notice/wpcom-sidebar-notice.php';
 		require_once __DIR__ . '/features/wpcom-themes/wpcom-themes.php';
 
+		// We include WPCom Themes results and installation on WoA sites only and non-WP_CLI context.
```

Comment:
```suggestion
		// We include WPCom Themes results and installation on WoA sites only and non-WP_CLI context. This is because we want to retain the core themes behaviour in the CLI context.
		// For now we don't want this to run on Simple sites, although the plan is to use this code to match the Simple and Atomic experience using these classes.
```